src/test/java/com/github/cameltooling/lsp/internal/completion/modeline/CamelKModelineCamelOpenAPITest.java

```diff
@@ -93,14 +93,5 @@ private File createFileStructureForTest() throws IOException {
 		anotherFile.createNewFile();
 		return camelKfile;
 	}
-	
-	@Test
-	void testNoExceptionWithPartialURI() throws Exception {
-		CamelLanguageServer camelLanguageServer = initializeLanguageServer("# camel-k: resource=", ".yaml");
-		
-		CompletableFuture<Either<List<CompletionItem>, CompletionList>> completions = getCompletionFor(camelLanguageServer, new Position(0, 20));
-		
-		List<CompletionItem> completionItems = completions.get().getLeft();
-		assertThat(completionItems).isEmpty();
-	}
```

Comment:
I think it was a leftover of a refactor. This usecase is completely covered by other tests